### PR TITLE
[Enhancement] (nereids)implement showQueryProfileCommand in nereids

### LINKED ISSUE #42772: [Enhancement] (nereids)implement showQueryProfileCommand in nereids
showQueryProfile command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following showQueryProfile command in nereids:
unsupportedshowQueryProfileStatement
``` SHOW QUERY PROFILE queryIdPath=STRING_LITERAL                                 ```

### PR DESCRIPTION
**Pull request by rijeshkp**
### What problem does this PR solve?

Issue Number: close #42772

Related PR: #xxx

Problem Summary:

 Support showQueryProfile command in nereids.

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java [hunk 1]
```diff
@@ -5014,5 +5016,11 @@ public LogicalPlan visitAdminShowTabletStorageFormat(AdminShowTabletStorageForma
     public LogicalPlan visitShowTabletStorageFormat(ShowTabletStorageFormatContext ctx) {
         return new ShowTabletStorageFormatCommand(ctx.VERBOSE() != null);
     }
+
+    @Override
+    public LogicalPlan visitShowQueryProfile(ShowQueryProfileContext ctx) {
+        String queryIdPath = ctx.queryIdPath.getText();
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java:5022]:
```suggestion
        String queryIdPath = stripQuotes(ctx.queryIdPath.getText());
```

**Comment by rijeshkp** [comment]
run buildall

**Comment by rijeshkp** [comment]
run buildall

**Comment by rijeshkp** [comment]
run buildall

**Review by Vallishp — APPROVED** [review]
LGTM

**Review by starocean999 — APPROVED** [review]

**Pull request merged**